packages/error/src/index.ts

```diff
@@ -0,0 +1,21 @@
+const isProduction: boolean = process.env.NODE_ENV === "production";
+const prefix = "\nVisit https://community.frontity.org for help! 🙂\n";
+
+// Throw an error if the condition fails
+// Strip out error messages for production
+// > Not providing an inline default argument for message as the result is smaller
+export default function invariant(condition: boolean, message?: string) {
+  if (condition) {
+    return;
+  }
+  // Condition not passed
+
+  if (isProduction) {
+    // In production we strip the message but still throw
```

Comment:
I really like the idea of using `babel-plugin-dev-expression` but if we are using it, why do we need to strip the message here? https://www.npmjs.com/package/babel-plugin-dev-expression#invariant

Reply:
So, as far as I understand, the use case is something like in React: In dev you get a full error message, but in production you just get an error code which you can look up to see what the full error message is. Maybe it's a bit overengineered 😅. We don't actually need to use it to strip out whatever is in `__DEV__` I realized.